salt/pillar/profiles.sls

```diff
@@ -22,6 +22,10 @@ profiles:
         level: DEBUG
     orcid_dummy:
         pinned_revision_file: /srv/profiles/orcid-dummy.sha1
+    sns:
+        name: bus-profiles
```

Comment:
this name (when tied together with the `--$env` suffix) is the correct one, but you should correct https://github.com/elifesciences/profiles-formula/blob/master/salt/profiles/init.sls#L136 to use this, that name was a placeholder

Reply:
OK have updated the name.